upi/aws/cloudformation/06_cluster_worker_node.yaml

```diff
@@ -37,15 +37,15 @@ Parameters:
     - "m4.xlarge"
     - "m4.2xlarge"
     - "m4.4xlarge"
-    - "m4.8xlarge"
+    - "m4.10xlarge"
```

Comment:
`m4.10xlarge` is already in the list

Reply:
@patrickdillon, thank you! Will you PTAL?